modules/ui/issues_info.js

```diff
@@ -29,7 +29,8 @@ export function uiIssuesInfo(context) {
         var liveIssues = context.validator().getIssues({
             what: prefs('validate-what') || 'edited',
             where: prefs('validate-where') || 'all'
-        });
+        }).filter(issue => issue.severity !== 'suggestion');
```

Comment:
I wonder if we should make the table from https://github.com/openstreetmap/iD/pull/11020#issue-3039567699 an object in code and use that for filters like this one. This would move the documentation in code, which I quite like…

I am thinking about something like…

```
const showIssue = {
 preventChanges: { suggestion: false, warning: false, error: true },
 showOnChangesetPage: { suggestion: false, warning: true, error: true },
 countInChangesetTag: { suggestion: false, warning: true, error: true },
 etc: { suggestion: false, warning: false, error: true },
}
```

So this filter would become something like `.filter(issue => showIssue.showOnChangesetPage[issue.severity])`